packages/components/src/composite/legacy/test/index.tsx

```diff
@@ -0,0 +1,599 @@
+/**
+ * External dependencies
+ */
+import { queryByAttribute, render, screen } from '@testing-library/react';
+import { press, waitFor } from '@ariakit/test';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Composite,
+	CompositeGroup,
+	CompositeItem,
+	useCompositeState,
+} from '..';
+
+// This is necessary because of how Ariakit calculates page up and
+// page down. Without this, nothing has a height, and so paging up
+// and down doesn't behave as expected in tests.
+
+jest.spyOn( HTMLElement.prototype, 'clientHeight', 'get' ).mockImplementation(
+	function getClientHeight( this: HTMLElement ) {
+		if ( this.tagName === 'BODY' ) {
+			return window.outerHeight;
+		}
+		return 50;
+	}
+);
```

Comment:
Do we need to restore this mock ? From a quick search, I couldn't see the [`clearMocks` flag](https://jestjs.io/docs/configuration#clearmocks-boolean) set to `true` to automatically reset mocks between each test suite.